### ISSUE TITLE
LaTeX does not render when selfHosted

### ISSUE THREAD

**Issue by pmeckoni** [halogenica/beautifulhugo#393]
When I use `\(\LaTeX\)`, it does not render and instead gives an error in the browser console inspector.

``` config.toml
[Params]
 selfHosted = true
```

However everything works well when I use 

``` config.toml
[Params]
 selfHosted = false
```

**Comment by pmeckoni** [comment]
Additionally, I found that other latex environments also don't work (like the below sample doesn't work):
``` sample.tex
\begin{align} 
x &= y
\end{align}
```
I found the problem is related to the old version of KaTeX. The version available in `layouts/partials/footer.html` is referring to 0.10.0 which was released in 2018. The current KaTeX version is 0.13.11.

While upgrading to the latest version of KaTeX may be a solution, please try to move from KaTeX to MathJax. 
MathJax 2.x is superior to KaTeX, though slower. When MathJax 3 is completely released, we expect it to be faster than KaTeX. [source](https://www.intmath.com/cg5/katex-mathjax-comparison.php).

So summarizing the issue:
1. KaTeX from CDN needs to be upgraded.
2. KaTeX on `selfHosted` needs to be upgraded.
3. Consider moving from KaTeX to MathJax.

**Comment by VincentTam** [comment]
![rendered by MathJax 3](https://user-images.githubusercontent.com/5748535/172054755-c8d5a310-3c7c-4ddb-81ed-ff9282f68695.png)
![rendered by KaTeX](https://user-images.githubusercontent.com/5748535/172054809-daf4cce4-9c77-4b12-9e5f-ce4cce5776e2.png)
I tested this with Firefox on Xubuntu but KaTeX was a bit faster.

**Comment by pmeckoni** [comment]
Yes KaTeX is faster. MathJax is more complete. Perhaps one way out is to provide both and let users to choose KaTeX or MathJax
